src/ds/ts/create-alter.md

```diff
@@ -0,0 +1,58 @@
+These tutorials will demonstrate Redis's' ability to store time series data using the bike shop use case.
+
+The bike shop company consists of multiple physical stores and an online presence. It would be helpful to have an aggregate view of sales volume across all the stores.
+
+### Create a time series
+
+You create time series using the `TS.CREATE` command. `TS.CREATE` requires one argument, the `key`, and can also take several optional arguments. These are:
+
+- `RETENTION period` - where `period` is the maximum age for samples compared to the highest reported timestamp in milliseconds. When set to 0, which is the default, samples never expire.
+- `ENCODING enc` - where `enc` is one of `COMPRESSED` (default) or `UNCOMPRESSED`. `COMPRESSED` is almost always the right choice. Compression not only saves memory, as much as a 90% reductions, but also improves performance due to fewer required memory accesses. Exceptions include: highly irregular timestamps or values.
+- `CHUNK_SIZE size` - where `size` is the initial allocation size in bytes for the data part of each new chunk. Actual chunks may consume more memory. Changing chunk size using TS.ALTER does not affect existing chunks. `size` must be a multiple of 8 in the range 48 to 1048576 (1 MB). The default size is 4096, representing a single page of memory.
+- `DUPLICATE_POLICY policy` -  defines the policy for handling insertion of multiple samples with identical timestamps. See the `TS.CREATE` manual page for the complete list of policies. The default policy is `BLOCK`, which means that Redis will ignore any newly reported values and reply with an error.
+- `LABELS label value [label value...]` - a set of label-value pairs that represent metadata labels for the key and serve as a secondary index.
+
+The following example creates a time series key for each of five bike shops that will track the total sales for each. Each key has an appropriate region label, `east` or `west`, and also a label indicating that the time series is not compacted. The labels allow you to query bike sales performance over specific periods on a per-shop or per-region basis.
+
+```redis Create time series for each shop
+TS.CREATE bike_sales_1 DUPLICATE_POLICY SUM LABELS region east compacted no
```

Comment:
I think better names would be `bike_sales:1` etc to follow the key name convention (see https://stackoverflow.com/a/6971415/171006)